go.mod

```diff
@@ -1,23 +1,21 @@
 module github.com/go-pkgz/jrpc
 
-go 1.23
+go 1.23.0
```

Comment:
accidentall, i think?

Reply:
It's go mod tidy result, if I remove it it's getting back after the run.

Reply:
i think pinning version like this will cause issues to anyone with version .2 or .3

> It's go mod tidy result, if I remove it it's getting back after the run.

This is likely because you use v1.24 for this project. If you switch GOROOT or whatever it is called in GoLand to 1.23, it will happily accept it.